Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM ubuntu:14.04.4
+FROM ubuntu:14.04
```

Comment:
Correct. 16 has been out for awhile now and seems pretty stable. Any reason to stay on 14?